meson.build

```diff
@@ -0,0 +1,112 @@
+project(
+    'pyconcrete',
+    'c',
+    meson_version: '>= 1.1',
+)
+
+INSTALL_DIR = 'pyconcrete'
+PYCONCRETE_EXE_NAME = 'pyconcrete'
+BUILD_UTILITY_DIR = './meson_utility/'
+
+version = run_command('cat', 'VERSION', check: true).stdout().strip()
+python_ins = import('python').find_installation(pure: false)
+python_ver = python_ins.language_version()
+
+
+#----------------------------
+# secret_key.h
+#----------------------------
+
+passphrase = get_option('passphrase')
+if passphrase == ''
+    error('Must assign a passphrase to build pyconcrete')
+endif
+
+prog_python = find_program('python')
```

Comment:
`python_ins` is already a found python program, you don't need to look for another. Furthermore, looking directly for "python" assumes that python is a symlink to python3, which isn't universally the case.

Reply:
I just realized that python_installation could replace find_program() & dependency(), thanks.

But I have a question, how does meson find the python installation if the platform have more than 1 python version? How can I assign the correctly python version which I want to build with the embedded package?

Reply:
When using meson via the meson-python build backend, it passes a "machine file" that overrides the detected python3 used by find_installation to the one that is being used by pep 517/518 (that is to say, the one you are invoking `pip install` or `python -m build`.

If you're invoking `meson setup` directly then use the following machine file:
```ini
[binaries]
python = '/path/to/preferred/python's]
```

via the `--native-file=XXX` argument to meson